tt_metal/hw/inc/grayskull/dev_mem_map.h

```diff
@@ -87,8 +87,8 @@
 /////////////
 // Stack info
 // Increasing the stack size comes at the expense of less local memory for globals
-#define MEM_BRISC_STACK_SIZE 768
-#define MEM_NCRISC_STACK_SIZE 768
+#define MEM_BRISC_STACK_SIZE 752
+#define MEM_NCRISC_STACK_SIZE 752
```

Comment:
Just curious, how did this happen? Were we always reserving more stack space than we needed?

Reply:
I added something to global space, made globals overflow into the stack space, so I had to shrink the stack space.  We have no data on how big stack should be, these values are inherited from buda.  Next up on @tt-dma todo list is to add stack sanitization to watcher so we can resize this w/ ~ confidence

Reply:
should this be applied to WH and BH?

Reply:
hmm, everything passes on WH, I assumed that the size of the L1 cache was different there but its not.  Not sure what is using more memory on GS vs WH.  It is possible this breaks BH, I'll give @abhullar-tt a heads up

Reply:
I think the reason we hit this on GS and not WH is because the bank noc address tables are bigger on GS (maybe?)